src/cache/cache_las.c

```diff
@@ -787,6 +788,9 @@ __wt_las_insert_block(WT_CURSOR *cursor,
 		__las_insert_block_verbose(session, btree, multi);
 	}
 
+	WT_RET(__wt_block_manager_named_size(session, WT_LAS_FILE, &las_size));
+	WT_STAT_CONN_SET(session, cache_lookaside_ondisk, las_size);
+
```

Comment:
Do we want to do this in the case of an error? I think we probably do not. I'd move it above the `err` label and change the `WT_RET` to `WT_ERR`.

Reply:
1573b8f

Makes sense. I've moved it down and used `WT_ERR`.